src/main/scala/com/softwaremill/react/kafka/KafkaActorSubscriber.scala

```diff
@@ -23,17 +22,12 @@ private[kafka] class KafkaActorSubscriber[T](
       processElement(element.asInstanceOf[T])
     case ActorSubscriberMessage.OnError(ex) =>
       handleError(ex)
-    case ActorSubscriberMessage.OnComplete =>
+    case ActorSubscriberMessage.OnComplete | "Stop" =>
```

Comment:
Sorry, yeah with Akka I meant akka-streams/the reactive stream spec. And I meant if implementations really have to add this kind of (indeed smelly) things to their production code then it is something worth thinking about/solving in akka-streams (or even the spec).